common/interfaces/desktop_app_interface/usb_rx_parser.c

```diff
@@ -0,0 +1,296 @@
+/**
+ * @file    usb_rx_parser.c
```

Comment:
consider renaming to `comm_rx_parser.c` as it is independent of physical layer

Reply:
not valid because we are calling lusb functions, therefore this comment is redundant